src/handle.ml

```diff
@@ -74,6 +74,7 @@ let handle_cmd_aux : sig_state -> command -> sig_state * proof_data option =
         | Sym_inj   :: [] -> Injec
         | _               -> fatal cmd.pos "Multiple symbol tags."
       in
+      let implicits = Scope.get_params_implicitness_of_type a in
```

Comment:
Please, move this just before line 88: let s = Sign.add_symbol ...

Reply:
If I do that then I need to change the lines in between because 'a' is being shadowed. I personally don't mind, but I know you like changes to be minimalist.

Reply:
I asked you this on purpose: implicits should be computed on xs and a!